conans/cli/api/subapi/install.py

```diff
@@ -38,9 +42,64 @@ def install_consumer(deps_graph, generators=None, source_folder=None, output_fol
         conanfile.folders.set_base_generators(output_folder)
         conanfile.folders.set_base_build(output_folder)
 
+        _do_deploys(self.conan_api, conanfile, deploy, output_folder)
+
         # Add cli -g generators
         conanfile.generators = list(set(conanfile.generators).union(generators or []))
         write_generators(conanfile)
 
         if type(conanfile).system_requirements != ConanFile.system_requirements:
             call_system_requirements(conanfile)
+
+
+def _do_deploys(conan_api, conanfile, deploy, output_folder):
+    # Handle the deploys
+    cwd = os.getcwd()
+    for d in deploy or []:
+        # First, find the file containing the deployer, could be local or in the cache
+        if d.startswith("conan"):  # built-in!
+            deployer = {"conan_full_deploy": conan_full_deploy}[d]
```

Comment:
what about if instead of using `conan_full_deploy` we'd use `default`? I think it's enough to put `default` to suppose that we want to use the Conan one, isn't it?

Reply:
Indeed, you can specify it in the docs, but I think it could be important to put a couple of examples here just to demonstrate what are the most common values.